app/saqib/page.js

```diff
@@ -3,14 +3,15 @@
 import React, { useState } from "react";
 
 export default function TodoApp() {
+  const [newTask, setNewTask] = useState(" "); // for new todo task from input
+
   const [todos, setTodos] = useState([
     { id: 1, task: "Study js", completed: false },
     { id: 2, task: "complete todo list react", completed: false },
-    { id: 3, task: "Done the shopping", completed: true },
+    { id: 3, task: "Done the shopping", completed: false },
   ]);
 
-  const [newTask, setNewTask] = useState(" ");
-
+  // this function will add new task to the list
   function addTask(e) {
     e.preventDefault();
     const newTodo = {
```

Comment:
the `id` here is actually causing the error, sorry I gave you the wrong logic. You can copy Saliha's logic for this!